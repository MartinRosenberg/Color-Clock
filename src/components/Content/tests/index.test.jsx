import { shallow } from 'enzyme'
import React from 'react'
import { Content } from '../index'

describe('Content component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<Content/>)
    expect(wrapper).toMatchSnapshot()
  })
})
