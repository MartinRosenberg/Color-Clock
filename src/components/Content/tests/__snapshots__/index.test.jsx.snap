// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component should render correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "height": "100vh",
      "justifyContent": "center",
      "width": "100vw",
    }
  }
/>
`;
